code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -262,10 +262,10 @@
 	if(.)
 		for(var/obj/item/mop/cyborg/M in R.module.modules)
 			R.module.remove_module(M, TRUE)
-
-	var/obj/item/mop/advanced/cyborg/A = new /obj/item/mop/advanced/cyborg(R.module)
-	R.module.basic_modules += A
-	R.module.add_module(A, FALSE, TRUE)
+		//Replace with M with advanced mop
```

Comment:
This comment still doesn't make sense in english

Reply:
M is the var name of the mop module that is removed in the line above